### PR TITLE
Fix/acccount 403 login screen

### PR DESCRIPTION
**Pull request by andrewkikot**
This PR solves issues with login, privileges and redirects in cases such as when there is a history url to admin resource, but user does not  have such privileges (and does not  have dashboard privileges either)  or user does not have any privileges at all...

### CODE REVIEW

File: src/app/home/home-default/home-default.component.html [hunk 1]
```diff
@@ -0,0 +1,8 @@
...
...mVyc2lvbj0iMS4wIiBlbmNvZGluZz0idXRmLTgiPz48c3ZnIHZlcnNpb249IjEuMSIgaWQ9IkxheWVyXzEiIHhtbG5zPSJodHRwOi8vd3d3LnczLm9yZy8yMDAwL3N2ZyIgeG1sbnM6eGxpbms9Imh0dHA6Ly93d3cudzMub3JnLzE5OTkveGxpbmsiIHg9IjBweCIgeT0iMHB4IiB2aWV3Qm94PSIwIDAgMjAzIDIwMy4zIiBzdHlsZT0iZW5hYmxlLWJhY2tncm91bmQ6bmV3IDAgMCAyMDMgMjAzLjM7IiB4bWw6c3BhY2U9InByZXNlcnZlIj48c3R5bGUgdHlwZT0idGV4dC9jc3MiPi5zdDB7ZmlsbDojQ0VDQ0NDO308L3N0eWxlPjxwYXRoIGNsYXNzPSJzdDAiIGQ9Ik0xNzMsNzIuNFYzMC45aC00MS41TDEwMi4yLDEuNUw3Mi44LDMwLjlIMzEuM3Y0MS41TDIsMTAxLjdMMzEuMywxMzF2NDEuNWg0MS41bDI5LjMsMjkuM2wyOS4zLTI5LjNIMTczVjEzMWwyOS4zLTI5LjNMMTczLDcyLjR6IE0xMDIuMiwxNjIuNmMtMzMuNiwwLTYwLjktMjcuMy02MC45LTYwLjlzMjcuMy02MC45LDYwLjktNjAuOVMxNjMsNjguMSwxNjMsMTAxLjdTMTM1LjgsMTYyLjYsMTAyLjIsMTYyLjZ6Ii8+PGc+PHBhdGggY2xhc3M9InN0MCIgZD0iTTEwMC4zLDEzNmMtMjYuMSwwLTMzLjYtMjIuOS0zMy43LTIzLjFjLTAuNS0xLjYsMC40LTMuMywyLTMuOGMxLjYtMC41LDMuMywwLjQsMy44LDEuOWMwLjMsMC44LDYuNCwxOC45LDI4LDE4LjljMjEuNSwwLDI2LjYtMTgsMjYuOC0xOC44YzAuNC0xLjYsMi4xLTIuNSwzLjctMi4xYzEuNiwwLjQsMi42LDIuMSwyLjEsMy43QzEzMi44LDExMywxMjYuNSwxMzYsMTAwLjMsMTM2eiIvPjwvZz48Y2lyY2xlIGNsYXNzPSJzdDAiIGN4PSI4NSIgY3k9IjkxLjUiIHI9IjUuNSIvPjxjaXJjbGUgY2xhc3M9InN0MCIgY3g9IjExNS4xIiBjeT0iOTEuNSIgcj0iNS41Ii8+PC9zdmc+"
```
Comment (major13ua) [inline src/app/home/home-default/home-default.component.html:3]: May be it is better to move data:... to .scss?

File: src/app/home/home-default/home-default.component.ts [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+}
+
+@Component({
+    selector: 'xm-home-default',
+    templateUrl: './home-default.component.html',
+    styleUrls: ['./home-default.component.scss'],
+})
+export class HomeDefaultComponent implements OnInit {
+
+    public homePageOptions: IHOmePageOptions;
```
Comment (major13ua) [inline src/app/home/home-default/home-default.component.ts:19]: Spelling mistake IHOme --> IHome

File: src/app/home/home.component.ts [hunk 3]
```diff
@@ -83,6 +69,29 @@ export class HomeComponent implements OnInit, OnDestroy {
         return this.principal.isAuthenticated();
     }
 
+    private getConfigAndNavigate(): void {
```
Comment (major13ua) [inline src/app/home/home.component.ts:72]: Can we re-write this part as template subscription.
So **getConfigAndNavigate** will be triggered from || async pipe from template?

File: src/app/home/home-default/home-default.component.html (hunk 1 above)
Comment (andrewkikot) [reply src/app/home/home-default/home-default.component.html]: done

**Pull request merged**